enso/experiment/__init__.py

```diff
@@ -196,7 +204,19 @@ def _run_experiment(self, dataset_name, current_setting, experiments):
                             time.sleep(0.1)
 
         return results
-
+    
+    @staticmethod
+    def _get_results_row(name, score, train_score, internal_setting, train_key, test_key):
```

Comment:
The argument order in this function hurts my brain.  Could we at least keep train / test scores in the same order as train / test keys?